python/tests/test_options.py

```diff
@@ -12,13 +12,20 @@ def test_defaults_to_win10_chrome_latest(self):
         assert sauce.browserVersion == 'latest'
         assert sauce.platformName == 'Windows 10'
 
-    def test_accepts_browser_version_platform_name(self):
+    def test_accepts_browser_name(self):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/app/SauceLabs/simple_sauce/pullRequest?prid=4626732)